slack-moji/models/user.rb

```diff
@@ -67,14 +67,19 @@ def to_s
     "user_id=#{user_id}, user_name=#{user_name}"
   end
 
-  def emoji_count?
-    !emoji_count.nil? || emoji_count == 0
+  def not_updating_status?
+    status == :unsubscribed
+  end
+
+  def using_emoji_status?
+    status == :emoji
   end
 
   def emoji_text
-    case emoji_count
-    when 0 then 'No Emoji'
-    else "#{emoji_count} Emoji"
+    if using_emoji_status?
```

Comment:
Turn this into 

```ruby
case status
when :off
when :emoji
else 
end
```

This way you don't need those helpers like `using_emoji_status?`. Those will proliferate.